modules/cnv-deleting-vms.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-delete-vms.adoc
+
+[id="cnv-deleting-vms_{context}"]
+
+= Deleting a virtual machine and PVCs using the CLI
+
+When you delete a virtual machine, the PVC it uses is unbound. If you
+do not plan to bind this PVC to a different VM, delete it, too.
+
+You can only delete objects in the project you are currently working in,
+unless you specify the `-n <project_name>` option.
+
+.Procedure
+
+Run these commands to delete the virtual machine and the PVC.
```

Comment:
Single step procedures need to use * on the single steps.

From the [mod docs guide](https://redhat-documentation.github.io/modular-docs/): 
> For single-step procedures, use an unnumbered bullet instead of a numbered list.

Reply:
Done.